docs/root/version_history/current.rst

```diff
@@ -24,6 +24,9 @@ Minor Behavior Changes
   ``use_unsigned_payload`` filter option (default false).
 * cluster: added default value of 5 seconds for :ref:`connect_timeout <envoy_v3_api_field_config.cluster.v3.Cluster.connect_timeout>`.
 * dns cache: the new :ref:`dns_query_timeout <envoy_v3_api_field_extensions.common.dynamic_forward_proxy.v3.DnsCacheConfig.dns_query_timeout>` option has a default of 5s. See below for more information.
+* grpc stream: reduced log level for "Unable to establish new stream" to debug.
```

Comment:
Can you reflow these lines? I believe we tend to keep these as single lines, only broken up once the 100 char limit per line is hit